### PR TITLE
PUT /api/v1/playlists/:id 

### PR DESCRIPTION
**Pull request by alect47**
User can update a specific playlist title
```
PUT /api/v1/playlists/1
body: 
{
  "title": "Updated Title",
}
```
- Updates playlist title
- Successful response status of 200
- Sad path message if invalid id, missing title or title that already exists and status of 404

expected response:
```
  {
    "id": 1,
    "title": "Updated Title",
    "createdAt": 2019-11-26T16:03:43+00:00,
    "updatedAt": 2019-11-26T16:03:43+00:00
  }
```

### CODE REVIEW

File: tests/requests/playlist.spec.js [hunk 1]
```diff
@@ -107,8 +112,52 @@ describe('Test GET api/v1/playlists', () => {
       expect(res.statusCode).toBe(404)
       expect(res.body.error).toBe("No playlists found")
     })
+});
 
-  afterEach(() => {
-    database.raw('truncate table playlists cascade');
+describe('Test PUT api/v1/playlists/:id', () => {
+  beforeEach(async () => {
```
Comment (hsmitha26) [inline tests/requests/playlist.spec.js]: Excellent edge case testing.

File: routes/api/v1/playlists.js [hunk 2]
```diff
@@ -40,6 +40,36 @@ router.post('/', (request, response) => {
     }
 });
 
+router.put('/:id', (request, response) => {
```
Comment (hsmitha26) [inline routes/api/v1/playlists.js]: Functionality looks great. Hopefully, we can refactor to eliminate multiple database calls next.

**Comment by hsmitha26** [comment]
Resolved merge conflicts.  Ready for staging.
